### ISSUE TITLE
system.describe example

### ISSUE THREAD

**Issue by brandonmwest** [jcubic/jquery.terminal#322]
When using JSON-RPC, the client makes a call to system.describe; can you provide an example of a system.describe response?

**Comment by jcubic** [comment]
It was in json-rpc spec when I created first version of jQuery Terminal but I can't find it anymore. It should look like this:
```json
{ 
  "sdversion": "1.0",
  "name": "DemoService",
  "address": "http:\/\/localhost\/projects\/jcubic\/leash\/index.php",
  "id":"urn:md5:4e39d82b5acc6b5cc1e7a41b091f6590",
  "procs" :[
    {"name":"echo","params":["string"]}
  ]
}
```

Only `name`, `id` and `procs` are required and prams can be any array only length is checked and it will be ignored when checkArity is option is set to false.

Just notice that it had hardcoded DemoService when fetching data from system.describe LOL, removed it now it will be in version 1.5 just will be released soon. It was added when I've added validation for JSON-RPC.

Need to update docs.
